### ISSUE TITLE
indexedDB inside MountableFileSystem require options

### ISSUE THREAD

**Issue by jcubic** [jvilk/BrowserFS#243]
I had code like this:
```javascript
BrowserFS.configure({
    fs: 'MountableFileSystem',
    options: {
        '/': { fs: 'IndexedDB'},
        '/tmp': { fs: 'InMemory' }
    }   
    //fs: 'IndexedDB', options: {}
}, function(err) {

});
```
and it throw exception somewhere from browserfs, if I add opitons for  indexedDB the error go away.

It should be documented that options is required.

**Comment by jvilk** [comment]
Thanks for the note!

**Comment by Syndace** [comment]
I suggest to fix the example in the README.md file ASAP, it's not cool if the very first example new users might run already throws an exception.

Also this is probably a bug and not a documentation issue.
Looking at the notes for release 1.4.3:

> Options object is optional for Create() methods when file system takes no options. Previously, the behavior was inconsistent. Now, calling Create(cb) is equivalent to calling Create({}, cb).

The exact error message for me is
> Uncaught TypeError: Cannot read property 'storeName' of undefined

which to me sounds as if `undefined` gets passed instead of `{}`.

**Comment by Mastaleru** [comment]
This bug is still reproducible at the time I'm writing this.

**Comment by pyramation** [comment]
fwiw, still running into this

**Comment by codenamezjames** [comment]
This is still an issue

**Comment by emeryberger** [comment]
Bumping @jvilk

**Comment by Project-Magenta** [comment]
Bumping @jvilk

**Comment by killinux** [comment]
use this 
```javascript
options: {
      "/zip": {
        fs: "ZipFS",
        options: {
          // Wrap as Buffer object.
          zipData: Buffer.from(zipData)
        }
      },
      "/tmp": { fs: "InMemory" },
      "/home": { fs: "IndexedDB",
        options: {
           storeName : "mydata"
        }
      }
    }
```
add 
```javascript
options: {
           storeName : "mydata"
}
```

**Comment by jcubic** [comment]
@killinux thanks, it was few years ago but I still have project that use this library and it will be useful to have `/tmp` in memory.

**Comment by jcubic** [comment]
@killinux Your comment make no sense, in the original post I've added that options are required and it's not documented.

**Comment by soren-mou-jakobsen** [comment]
Still an issue

**Comment by dr-vortex** [comment]
Fixed in #352
